packages/edit-site/src/components/create-template-part-modal/index.js

```diff
@@ -60,14 +60,12 @@ export default function CreateTemplatePartModal( {
 	);
 
 	async function createTemplatePart() {
-		if ( ! title ) {
-			createErrorNotice( __( 'Please enter a title.' ), {
-				type: 'snackbar',
-			} );
```

Comment:
This was a death path; the `createTemplatePart` was never called when the `title` was empty, so I removed it.